test/live.js

```diff
@@ -0,0 +1,4 @@
+export var value = 0;
+export function add(x) {
+  value += x;
```

Comment:
This is a known problem that I plan to solve by calling `module.export("value")` immediately after the assignment/update. For comparison, Babel would achieve this by replacing this assignment statement with `exports.value = value += x`.